.github/workflows/goreleaser-fleet.yaml

```diff
@@ -106,8 +106,8 @@ jobs:
         continue-on-error: true
         id: image_digests
         run: |
-          echo "digest_fleet=$(cat ./dist/artifact.json | jq -r '.[]|select(.type == "Published Docker Image" and (.name | contains("fleetdm/fleet:${{ steps.commit.outputs.short_commit }}"))) | select(. != null)|.extra.Digest')" >> "$GITHUB_OUTPUT"
```

Comment:
I'm sure I copy/pasted this from somewhere where it made sense, but in my test runs on Github it's definitely saving to `./dist/artifacts.json`, not `./dist/artifact.json`.